drivers/wifi/nrf700x/zephyr/src/qspi/src/rpu_hw_if.c

```diff
@@ -35,6 +35,14 @@ GPIO_DT_SPEC_GET(NRF7002_NODE, iovdd_ctrl_gpios);
 static const struct gpio_dt_spec bucken_spec =
 GPIO_DT_SPEC_GET(NRF7002_NODE, bucken_gpios);
 
+#if defined(CONFIG_BOARD_NRF7000DK_NRF5340_CPUAPP) || \
+	defined(CONFIG_BOARD_NRF7001DK_NRF5340_CPUAPP) || \
+	defined(CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP)
+#define NRF_RADIO_COEX_NODE DT_NODELABEL(nrf_radio_coex)
+static const struct gpio_dt_spec btrf_switch_spec =
+GPIO_DT_SPEC_GET(NRF_RADIO_COEX_NODE, btrf_switch_gpios);
+#endif /* CONFIG_BOARD_NRF700XDK_NRF5340_CPUAPP */
```

Comment:
Options `BOARD_NRF7000DK_NRF5340_CPUAPP` and `BOARD_NRF7001DK_NRF5340_CPUAPP` do not exist, so this won't work as expected.
But wouldn't it be better to compile this conditionally when the `btrf_switch_gpios` property is present? I mean something like:
```
#define NRF_RADIO_COEX_NODE DT_NODELABEL(nrf_radio_coex)
#if DT_NODE_HAS_PROP(NRF_RADIO_COEX_NODE, btrf_switch_gpios)
static const struct gpio_dt_spec btrf_switch_spec =
	GPIO_DT_SPEC_GET(NRF_RADIO_COEX_NODE, btrf_switch_gpios);
#endif
```